kani-driver/Cargo.toml

```diff
@@ -15,6 +15,7 @@ publish = false
 kani_metadata = { path = "../kani_metadata" }
 cargo_metadata = "0.18.0"
 anyhow = "1"
+chrono = "0.4.38"
```

Comment:
This crate seems to bring in tons of other dependencies as indicated by the `Cargo.lock` diff. Since all that is needed is to get the current time and print it, it the [`time` crate](https://crates.io/crates/time/0.3.36/dependencies) seems to be a lighter-weight version.

Reply:
Great suggestion! We're now using the `time` crate with the `formatting` feature.